### PR TITLE
Update moment and buffermetrics packages

### PR DESCRIPTION
**Pull request by albennett**
### Purpose
Saw a `We found a potential security vulnerability in one of your dependencies.` alert in buffer-publish repo for moment package. Updated moment package versions. 
### Notes
When I ran yarn, I also got an error `@bufferapp/buffermetrics@0.6.3-beta-06: The engine "node" is incompatible with this module.` I noticed the package version was updated recently, so I updated the versions in publish and the error went away. Was there a specific reason we were using 0.6.3-beta-06?

#### Staging Deployment

_This repo is CI/CD enabled and staging deployment should be available at:
https://<branch_name>.publish.buffer.com :smile:_

### CODE REVIEW

**Comment by hamstu** [comment]
👍 To the moment change.

Re; the buffermetrics notice. Not sure. Gonna ping @hharnisc on this one since it smells like something related to his final work on the grpc bugs we were seeing from that library. (I could be wrong on that.)  So let's wait for his input before we merge this (I can take care of it if you're away @albennett).

File: packages/modals/package.json [hunk 1]
```diff
@@ -11,7 +11,7 @@
   "dependencies": {
     "@bufferapp/keywrapper": "0.2.0",
     "@bufferapp/publish-upgrade-modal": "2.0.0",
-    "@bufferapp/buffermetrics": "0.5.2"
+    "@bufferapp/buffermetrics": "0.6.3"
```
Comment (hharnisc) [inline packages/modals/package.json]: Can you use `0.6.3-beta-06` for this :smile: should have everything from the `0.6.3` package and use native js driver

File: packages/server/package.json [hunk 2]
```diff
@@ -9,7 +9,7 @@
   "author": "hharnisc@gmail.com",
   "dependencies": {
     "@bufferapp/buffer-rpc": "0.0.4",
-    "@bufferapp/buffermetrics": "0.6.3-beta-06",
+    "@bufferapp/buffermetrics": "0.6.3",
```
Comment (hharnisc) [inline packages/server/package.json]: Can you use `0.6.3-beta-06` for this :smile: should have everything from the `0.6.3` package and use native js driver

File: packages/store/package.json [hunk 3]
```diff
@@ -12,7 +12,7 @@
     "@bufferapp/analyze-profile-selector": "0.65.0",
     "@bufferapp/app-sidebar": "1.9.24",
     "@bufferapp/async-data-fetch": "1.9.4",
-    "@bufferapp/buffermetrics": "0.6.3-beta-03",
+    "@bufferapp/buffermetrics": "0.6.3",
```
Comment (hharnisc) [inline packages/store/package.json]: same :smile_cat:

**Comment by hharnisc** [comment]
Thanks for the ping @hamstu we should be able to use `0.6.3-beta-06` for this (for now)

File: packages/store/package.json (hunk 3 above)
Comment (albennett) [reply packages/store/package.json]: Thanks! Upgrading my node version worked :)

**Pull request merged**